cython0.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: cython
+  version: 0.29.36
+  epoch: 0
+  description: Cython is an optimising static compiler for both the Python & the extended Cython programming languages.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - py3-setuptools
+      - python3-dev
+      - python3
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 702bbc80fe2e346c4ddc58f074e0054da56e0ced
+      repository: https://github.com/cython/cython
+      tag: 0.29.36
+
+  - runs: |
+      python setup.py build
+      python setup.py install --prefix=/usr --root="${{targets.destdir}}"
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: cython/cython
+    use-tag: true
+    tag-filter: 0.
```

Comment:
I'm getting the `-0` above from this scoping.

Reply:
Makes sense, just updated